app/src/main/java/org/fossasia/susi/ai/rest/responses/others/RelatedTopics.kt

```diff
@@ -0,0 +1,29 @@
+package org.fossasia.susi.ai.rest.responses.others
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+/**
+ * <h1>POJO class to parse related topics object in retrofit response from websearch client.</h1>
+
+ * Created by mayank on 09-12-2016.
+ */
+
+class RelatedTopics
+
+(@SerializedName("FirstURL")
+ @Expose
+ var url: String?,
+
+ @SerializedName("Text")
+ @Expose
+ var text: String?,
+
+ @SerializedName("Icon")
+ @Expose
+ var icon: WebIcon?) {
+
+    @SerializedName("Result")
+    @Expose
+    var result: String? = null
+}
```

Comment:
"It was just like that in java" does not mean it was right. If you don't need to initialize the above files I sent, then you don't need to initialize this one either. The values are automatically converted by Retrofit from the JSON object and assigned to variables. There is **no** need to explicitly initialize the variables. Hence we don't need a constructor.